unit_tests/test_sct_events_loaders.py

```diff
@@ -256,6 +256,46 @@ def test_cs_error_events_list(self):
         self.assertSetEqual(set(dir(CassandraStressLogEvent)) - set(dir(LogEvent)),
                             {ev.type for ev in CS_ERROR_EVENTS})
 
+    def test_cs_hint_in_flight_error(self):
+        def _get_event(line, expected_type, expected_severity):
+            for pattern, event in CS_ERROR_EVENTS_PATTERNS:
```

Comment:
if there are no events found for the line need to fail the test

Reply:
Added raise if event is not recognized